core/src/com/unciv/logic/civilization/Notification.kt

```diff
@@ -18,6 +19,24 @@ object NotificationIcon {
     const val Diplomacy = "OtherIcons/Diplomacy"
     const val City = "ImprovementIcons/City center"
     const val Citadel = "ImprovementIcons/Citadel"
+    const val Happiness = "StatIcons/Happiness"
+    const val Population = "StatIcons/Population"
+    const val CityState = "NationIcons/CityState"
+    const val Production = "StatIcons/Production"
+    const val Food = "StatIcons/Food"
+    const val Faith = "StatIcons/Faith"
+    
+    fun statToIcon(stat: Stat): String {
```

Comment:
I know Yair doesn't like it as much as I do but this is a prime example - for enum properties. The whole function completely unnecessary and replaced by direct access to the const & interned string in bytecode.